app/api/routes.py

```
@@ -401,4 +405,71 @@ def delete_record_route(
+        )
+    else:
+        # gemini 클라이언트 초기화 실패 시 500 에러 발생
+        raise HTTPException(status_code=500, detail="상담 에이전트 서비스 초기화에 실패했습니다. 서버 로그를 확인해주세요.")
+
+
+@router.post("/api/v1/consult/chat",
+     tags=["에이전트 상담"],
+     summary="에이전트 대화",
+     description="세션 ID를 사용하여 에이전트와 대화를 나눕니다."
+)
+def send_chat_message(request: ChatRequest, db: Session = Depends(get_db), current_user: User = Depends(AuthTokenDep)):
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)